src/tests/mock_spritecache.cpp

```diff
@@ -17,14 +17,14 @@
 
 static bool MockLoadNextSprite(int load_index)
 {
-	static Sprite *sprite = (Sprite *)AllocSprite(sizeof(*sprite));
+	void *ptr = AllocSprite(sizeof(Sprite));
```

Comment:
Hmm, this was specifically static to use the same buffer and avoid allocating thousands of dummy sprites, but of course that's not compatible when using a vector. Oh well, that doesn't really matter much. :)